.github/workflows/build.yml

```diff
@@ -79,7 +75,7 @@ jobs:
             ${{ env.CURL_CACHE_DIR }}/"${PACKER_ZIP}"
           sudo mv /usr/local/bin/packer /usr/local/bin/packer-default
           sudo ln -s /opt/packer/packer /usr/local/bin/packer
-      - uses: hashicorp/setup-terraform@v1
+      - uses: hashicorp/setup-terraform@v2
```

Comment:
Please update the other use of `hashicorp/setup-terraform`:
https://github.com/cisagov/skeleton-packer/blob/f5f8bb83bdb0cd9a13c8e6a71a75a302b1520315/.github/workflows/build.yml#L196

Reply:
Please see commit a8d6557.